Installing-and-Configuring/install-os.md

```diff
@@ -0,0 +1,326 @@
+
+Once Qubes has finished installing, you'll need to reboot and select the 'Boot
+ menu' option by hitting 'm'.
+
+Select the first boot option:
+
+```text
+1. Qubes, with Xen hypervisor [...]
+```
+
+Then make this the default boot entry by hitting 'd'.  This will also allow you
+ to seal the disk encryption key.
+
+You will need to input the disk recovery key here (almost for the last time),
+ and this should start the final stage of the Qubes installer.  Under
+ `Configure Qubes` you should select `Create USB qube holding all USB controllers`
+ so that they are protected from outside devices.  This step takes a little
+ while as the templates are configured...
+
+Eventually this will be done and you can click "Finish", then Qubes will
+give you a login screen with your login password.
+
+If you choose to add the disk encryption key to the TPM, you'll need to specify
+ which LUKS volume.  A default Qubes install will work if you leave the
+ 'Encrypted LVM group?' response blank and enter `/dev/sda2` when asked about
+ 'Encrypted devices?'.  For more details see the TPM Disk encryption keys
+ section below. You'll then be asked to enter the disk recovery key as well as
+ the new boot password you'll use to unseal that key.
+
+Once the key sealing process is complete, it will ask you to insert your GPG
+ card then enter your PIN to sign the config.  After this, it will reboot back
+ to the main boot menu.  Disconnect your GPG card otherwise Qubes might think
+ you have a USB keyboard.
+
+To start Heads now (and in the future), just hit 'y' for default boot.
+
+This should start the final stage of the Qubes installer.  Under
+'Configure Qubes' you should select `Create USB qube holding all USB controllers`
+ so that they are protected from outside devices.  This step takes a little
+ while as the templates are configured...
+
+Eventually this will be done and you can click "Finish", then Qubes will
+give you a login screen with your login password.
+
+After the first reboot, the boot entry will be different post-installation, so
+ after you hit 'y' to select default boot you will see a message:
+
+```text
+!!! Boot entry has changed - please set a new default
+```
+
+This will also happen on OS updates that changed the boot process (updating the
+   kernel or the initramfs, etc.).  If someone has tampered with your `/boot`
+   partition, this can also happen, so if you're not sure of the situation,
+   don't proceed.
+
+Choose the first option again ('1'), then make it the new default ('d'), confirm
+ that you're modifying the boot partition ('y'), and that you don't need to
+ reseal the disk key ('n').  You'll be asked to insert your GPG card and enter
+ the PIN to sign the new configs and the system will reboot and allow you to
+ proceed as normal.
+
+Installing extra software
+---
+
+dom0 probably has updates available. You'll want to install them before
+ switching `/` to read-only and signing the hashes:
```

Comment:
How you do that? Old artifact on RO root filesystem here too? Please remove here too.

Reply:
Removed.